hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/TestTags.java

```diff
@@ -47,4 +47,8 @@ private TestTags() {
      * ECDSA keys (whose signatures are inherently random).
      */
     public static final String NOT_REPEATABLE = "NOT_REPEATABLE";
+    /**
+     * Tags a test that can be run alone, without any other tests.
```

Comment:
"can" or "should" (or "will")?  After all, all of our tests "can" be run alone, without any other tests, if you just point at them and click "Run" ....

Reply:
It's mostly "will " 😄 . This is mostly grouping some tests with a tag, so they can run together and we can do validations for blocks streams. This can be deleted later, but seems a nice way to tests blocks are correct with expected outputs (same as in records)